src/ast/codegen_llvm.cpp

```diff
@@ -854,6 +859,7 @@ void CodegenLLVM::visit(Binop &binop)
 
       expr_ = b_.CreateStrcmp(left_string, right_string, inverse);
     }
+    expr_ = b_.CreateIntCast(expr_, b_.getInt64Ty(), false);
```

Comment:
It looks like `expr_` at this point holds a `Int8`. What was the reason to cast to `Int64`?